test/integration/targets/eos_eapi/tests/cli/zzz_reset.1

```diff
@@ -1,8 +1,9 @@
 - debug: msg="START CLI/RESET.YAML"
```

Comment:
This test will it get run anymore as it doesn't end in `.yaml`
If it's meant to be a note to return an EOS device to original configuration please add a comment saying that.